src/pinecone_client/vector_db.py

```diff
@@ -0,0 +1,54 @@
+import pinecone
+from langchain.embeddings.openai import OpenAIEmbeddings
+from dotenv import load_dotenv
+import os
+from langchain.vectorstores import Pinecone
+
+# 　環境変数の読み込み
+load_dotenv()
+INDEX_NAME = os.getenv("INDEX_NAME")
+PINECONE_API_KEY = os.getenv("PINECONE_API_KEY")
+PINECONE_ENV = os.getenv("PINECONE_ENV")
+
+if not INDEX_NAME:
+    raise ValueError("INDEX_NAME 環境変数が設定されていません。")
+if not PINECONE_API_KEY:
+    raise ValueError("PINECONE_API_KEY 環境変数が設定されていません。")
+if not PINECONE_ENV:
+    raise ValueError("PINECONE_ENV 環境変数が設定されていません。")
+
+# pinecone clientの初期化
+pinecone.init(api_key=PINECONE_API_KEY, environment=PINECONE_ENV)
```

Comment:
```suggestion
def _init_pinecone():
  pinecone.init(.....)
```
この辺りも関数化してあげると他の場所で使いやすくなりそうですね！